### ISSUE TITLE
Caffe doesn't overfit on the very small training dataset

### ISSUE THREAD

**Issue by LukasToma** [BVLC/caffe#1682]
Hello,

Thank you very much for designing caffe!

I have an issue with training a small CNN. During the training procedure the value of my loss function (which is softmax) drops quite easily from 0.70 down to 0.33-0.36. But then the loss on the training data stabilizes and starts to fluctuate in a very short segment (0.33, 0.36). I tried to increase the number of feature maps in convolutional layers, to incease the number of convolutional layers, to decrease the learning rate but nothing helps... The network stops to learn when the value of the loss function is in the vicinity of 0.35.

I have a very small training dataset (only 1500 images), so it seems to me that a big enough CNN should overfit quite quickly on it. Nevertheless, it doesn't happen and I don't understand why... Do you have any idea of what might be the reason for this problem?

Thank you in advance.

Grigory

**Comment by sguada** [comment]
All questions about usage, installation, code, and applications should be searched for and asked on the [caffe-users mailing list](https://groups.google.com/forum/#!forum/caffe-users).
